source/braille.py

```diff
@@ -1698,6 +1713,11 @@ def handleUpdate(self, obj):
 				break
 		else:
 			# No region for this object.
+			# There are some objects that may require an update even if they have no region.
+			# Late import to avoid circular import.
+			from NVDAObjects import NVDAObject
+			if isinstance(obj, NVDAObject) and obj.role == controlTypes.ROLE_PROGRESSBAR and obj.isInForeground:

```

Comment:
Currently, we don't show context for review, only for focus. Therefore, this is only useful when tethered to review. I'd suggest restricting it as such to avoid pointlessly walking parents (but the assumption should be explained in a comment).